resources/stsci.edu/schemas/bounding_box-1.0.0.yaml

```diff
@@ -0,0 +1,74 @@
+%YAML 1.1
```

Comment:
This and the compound bounding box need to have entries in the latest manifest document